ambry-clustermap/src/main/java/com.github.ambry.clustermap/CompositeClusterManager.java

```diff
@@ -23,6 +23,8 @@
 import java.util.List;
 import java.util.Set;
 
+import static com.github.ambry.clustermap.Datacenter.*;
```

Comment:
Now there is no dependency on `Datacenter` after the `LEGACY_DATACENTER_ID` is moved to `ClusterManagerUtils`.